### PR TITLE
Feature/form control changes 2

### PR DESCRIPTION
**Pull request by aamirhm**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests have been added (if needed)

### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-6740
### Change description ###
We have added a service dropdown for filter. 
**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts [hunk 1]
```diff
@@ -1,94 +1,114 @@
-import { Component, OnInit } from '@angular/core';
+import { Component } from '@angular/core';
 import { ActivatedRoute } from '@angular/router';
 import { FilterConfig } from '@hmcts/rpx-xui-common-lib/lib/models';
+import { CaseUserSearchErrorMessage } from 'src/cases/models/case.enum';
+import { StaffDataFilterService } from 'src/staff-administrator/services/staff-data-filter.service';
```
Comment (connorpgpmcelroy) [inline src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts:5]: Need to replace src with relative routes '../'

**Review by connorpgpmcelroy — CHANGES_REQUESTED** [review]
Just the src change needed and then i can approve

File: src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts (hunk 1 above)
Comment (connorpgpmcelroy) [reply src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts:5]: Also the build is failing

Comment (aamirhm) [reply src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts]: Well note and updated.

**Review by DavidJayakumar — APPROVED** [review]

File: src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts (hunk 1 above)
Comment (aamirhm) [reply src/staff-administrator/components/staff-adv-filter/staff-adv-filter.component.ts]: The build is failing as below PR should be merged before this (https://github.com/hmcts/rpx-xui-common-lib/pull/308).

**Pull request closed without merging**